portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,19 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomQuote() {
+  const officeQuotes =
+          [
+        '“Would I rather be feared or loved? Easy. Both. I want people to be afraid of how much they love me.” – Michael Scott',
+       '“I’m not superstitious, but I am a little stitious.” – Michael Scott',
+        '“And I knew exactly what to do. But in a much more real sense, I had no idea what to do.” – Michael Scott',
+         '"You miss 100% of the shots you dont take. - Wayne Gretsky" - Michael Scott'
```

Comment:
I think it would be good if we follow Google's js style guide https://google.github.io/styleguide/jsguide.html.

Can you fix the indentation to follow that:
https://google.github.io/styleguide/jsguide.html#formatting-array-literals

Also the often annoying missing trailing comma:
https://google.github.io/styleguide/jsguide.html#features-arrays-trailing-comma